localstack/services/s3/s3_utils.py

```diff
@@ -49,6 +49,9 @@
     'X-Amz-SignedHeaders', 'X-Amz-Signature'
 ]
 
+# headers to blacklist from request_dict.signed_headers
+BLACKLISTED_HEADERS = ['X-Amz-Security-Token']
```

Comment:
Looks great, Thanks for this PR @macnev2013 ! 👍 

nit: I'd rename this to something more descriptive, e.g., `PRESIGN_SKIPPED_HEADERS = ...` (not critical - can be done in a future PR..).

Reply:
Sure. I'll update it in follow up PR.